bin/truffleruby

```diff
@@ -93,7 +93,8 @@ if [ $on_graalvm = false ]; then
 
     if [ "$mx_time" -gt "$mvn_time" ]; then
         binary_truffle="$root/mx.imports/binary/truffle/mxbuild"
-        source_truffle="$(dirname "$root")/truffle/mxbuild"
+        source_truffle_suite_py=("$(dirname "$root")"/*/*/"mx.truffle/suite.py")
```

Comment:
The name of the repo is just not predefined. This is has already caused problems in the past and required workarounds from `mx` to try to use the "expected" name for that repo.

What about `*/truffle/mx.truffle/suite.py`? it should be much faster.

Also note that the version with 2 `*` is essentially what mx will do.